intel_owl/settings/django.py

```diff
@@ -8,7 +8,7 @@
     "certego_saas.ext.middlewares.StatsMiddleware",  # custom
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
-    "django.middleware.csrf.CsrfViewMiddleware",
+    # "django.middleware.csrf.CsrfViewMiddleware",
```

Comment:
this is to have the form working correctly, am I right? We should try to enable it again, I see if I can help in any way

Reply:
the postman had some issues with the APIs. i will enable it again. should work.